packages/api/src/timeseries/timeseries.ts

```diff
@@ -105,3 +105,45 @@ export interface TimeSeriesProperty<T extends number | string> {
     query?: TimeSeriesQuery,
   ) => AsyncGenerator<TimeSeriesPoint<T>>;
 }
+
+export interface GeotimeSeriesProperty<T extends GeoJSON.Point> {
```

Comment:
Lets make everything in here `readonly`.

Reply:
Should I do this to normal timeseries property as well?

Reply:
yeah